### ISSUE TITLE
Sofawiki 3.8.8 throws Fatal error

### ISSUE THREAD

**Issue by prbt2016** [bellenuit/sofawiki#25]
Hello,

I was in the process of manual installation of Sofawiki 3.8.8 on Centos 7 with PHP 5.6 to 7.4 , Apache 2.2. 

However, when I go to search (under search (both under singlesearchmenu and searchmenu)

After searching and submitting , the following error is thrown i.e : 

```
Fatal error: Call to undefined function  array_key_exists() in {{/{{PATH}}/{{TO}}/{{SOFAWIKI}}/inc/relation.php on line 898 
```

What could be the issue?.

Could you please replicate this issue at your end and fix this?.

**Comment by bellenuit** [comment]
Found error (invisible character after refactoring swRelation.
Fixed in 3.8.9

**Comment by prbt2016** [comment]
Hello @bellenuit ,

Thanks . It works fine now. Checked with 3.8.9.

Regards.
